templates/se2-docker/docker-compose.yml.tmpl

```diff
@@ -9,7 +9,7 @@ services:
       SE2_LOG_LEVEL: info
       SE2_HTTP_PORT: 8081
     env_file:
-      - SCC.env
+      - SE2.env
```

Comment:
Let's make sure that the file being generated is actually being renamed.

Reply:
This is coming from the `SE2.env.tmpl` in the same folder.